force-app/main/default/classes/PropertyService.cls

```diff
@@ -1,4 +1,4 @@
-public with sharing class PropertyService {
+public with sharing class PropertyService implements Database.Batchable<SObject>, Schedulable {
```

Comment:
don't make PropertyService batchable, create new class **PaymentCheckBatch**

Reply:
ok, got it! Thanks